src/media/css/tiles.styl

```diff
@@ -83,6 +85,9 @@ $greater-than-mobile = unquote('(min-width: 330px)');
         > div {
             margin-top: 5px;
         }
+        +rtl() {
+            margin-right: -10px;
```

Comment:
Is this twice the `margin-left` to account for it? Could this be `margin: 0 -5px -15px 0` instead?